### PR TITLE
Add information about the "Resume your last course" option

### PR DESCRIPTION
**Pull request by srpearce**
## [DOC-3892](https://openedx.atlassian.net/browse/DOC-3892)

Add information about the "Resume your last course" option. Adding info about resuming courses led to a reorg of two learner topics and minor changes to a course author guide topic.

### Date Needed (optional)

19 Mar 2018

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Subject matter expert: @sanfordstudent 
- [x] Doc team review (copy edit): @edx/doc
- [x] Product review: @shamck 
- [ ] Partner support: @jaakana
- [ ] Learner support: @apyersedx

### HTML Version

@sanfordstudent This is the only part that needs TR (it's two sentences): 
* [Return to the Most Recently Completed Unit in All Courses](http://draft-resume-your-last-course-option.readthedocs.io/en/latest/SFD_start_course.html#return-to-the-most-recently-completed-unit-in-all-courses)

@edx/doc @apyersedx The following topics need a copy edit and sanity check:
* [Starting a Course](http://draft-resume-your-last-course-option.readthedocs.io/en/latest/SFD_start_course.html)
* [Accessing Course Content and Information](http://draft-resume-your-last-course-option.readthedocs.io/en/latest/SFD_content_availability.html)

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/shared/students/SFD_start_course.rst [hunk 1]
```diff
@@ -30,152 +22,133 @@ lessons or assignments.
...
 
-You can also view the course and open its **Progress** page to view your final
-grade, as well as your grades on all course assignments.
+ When you complete a part of a course, a green check mark is visible on the
+ course outline. For more information, see :ref:`SFD Progress Indicators`.
 
-========================
-View Certificate Status
-========================
+Return to the Most Recently Completed Unit in All Courses
```
Comment (srpearce) [inline en_us/shared/students/SFD_start_course.rst:129]: @sanfordstudent Please review lines 132-137. Thanks!

Comment (sanfordstudent) [reply en_us/shared/students/SFD_start_course.rst:129]: Looks great, thanks @srpearce !

**Review by grantgoodmanedX — APPROVED** [review]
👍 Looks fine to me.

**Comment by apyersedx** [comment]
👍 looks good!

**Comment by shamck** [comment]
👍

**Pull request merged**